keyboards/yoichiro/lunakey_mini/config.h

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2020 Yoichiro Tanaka
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x5954  // "YT" - Yoichiro Tanaka
+#define PRODUCT_ID   0x0001
+#define DEVICE_VER   0x0001
+#define MANUFACTURER yoichiro
+#define PRODUCT      yoichiro Lunakey Mini
```

Comment:
```suggestion
#define PRODUCT      Lunakey Mini
```

Reply:
@fauxpark Thank you for the review comment. However, when I read the documentation of VIA, I found the following sentence:

> It is highly recommended that the value of PRODUCT is changed to be a combination of designer/vendor and the device name.

https://caniusevia.com/docs/configuring_qmk/#change-product

Therefore, I wrote the PRODUCT value as `MyName ProductName`. Should I remove my name and remain only the product name?

Reply:
@fauxpark I tried to find the product name like `DesignerName ProductName`, but I could not find such cases. I accept your suggestion and fix at 08df382cb.